src/app/app.component.ts

```diff
@@ -1,9 +1,24 @@
-import {Component, inject, OnInit} from '@angular/core';
+import {Component, HostListener, inject, OnInit} from '@angular/core';
 import {environment} from "../environment/environment";
 import {TranslateService} from "@ngx-translate/core";
 import {SocialShareSeoService} from "../common/cdk/social-share.seo.service";
 import {ChangeLanguageComponent} from "./component/change-language/change-language.component";
 import {NgOptimizedImage} from "@angular/common";
+import {NgIcon, provideIcons, provideNgIconsConfig} from "@ng-icons/core";
+import {bootstrapThreeDots, bootstrapXLg} from "@ng-icons/bootstrap-icons";
+
+enum MenuUseCase {
+  Desktop,
+  Mobile,
+  Both
+}
+
+interface MenuItem {
```

Comment:
Move the interface into file into interface folder and the file should have name: `i.menu-item.ts` and the interface should call IMenuItem

Reply:
done